cql3/prepared_statements_cache.hh

```diff
@@ -27,35 +27,27 @@ struct prepared_cache_entry_size {
 };
 
 typedef bytes cql_prepared_id_type;
-typedef int32_t thrift_prepared_id_type;
 
 /// \brief The key of the prepared statements cache
 ///
-/// We are going to store the CQL and Thrift prepared statements in the same cache therefore we need generate the key
-/// that is going to be unique in both cases. Thrift use int32_t as a prepared statement ID, CQL - MD5 digest.
-///
-/// We are going to use an std::pair<CQL_PREP_ID_TYPE, int64_t> as a key. For CQL statements we will use {CQL_PREP_ID, std::numeric_limits<int64_t>::max()} as a key
-/// and for Thrift - {CQL_PREP_ID_TYPE(0), THRIFT_PREP_ID}. This way CQL and Thrift keys' values will never collide.
+/// We are going to store the CQL prepared statements in the same cache therefore we need generate the key
```

Comment:
The phrase "same cache" now doesn't refer to anything ;-)

Reply:
i revised this comment to reflect the status quo. actually we should further simplify this class. but i want to leave it for another PR.